src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -192,11 +182,130 @@ public boolean rightLowerLimitReached() {
 
     @Override
     public void periodic() {
+        if (enabled) {
+            double output = verticalController.calculate(getLeftPosition());
+            double error = errorController.calculate(DriveTrain.getInstance().getPitch());
```

Comment:
I would store driveTrain as a private variable on this class so that we don't have to get instance more than once